integration-test/pipeline/rest-component-definition.js

```diff
@@ -83,19 +83,6 @@ export function CheckList() {
       "GET /v1beta/component-definitions?page_size=1&view=VIEW_FULL response component_definitions[0].connector_definition.spec is not null": (r) => r.json().component_definitions[0].connector_definition.spec !== null,
     });
 
-
-    // Fetch a page with operator definitions.
```

Comment:
It is correct to remove this test case. It was added on [this commit](https://github.com/instill-ai/pipeline-backend/commit/b8728c16483fb68cdaed65dff191d2500ec06e4e#diff-2b655ba613ca8386fd9f4c383200480492ee25711e24a575f3fc8b33531b4672R1915), where the component definition list was built from the individual connector and operator lists. In order to access the first operator we needed to know the size of the connector list. Once the database table was introduced and that we introduced featured components, this test became obsolete. Now we don't have a block of connectors and a block of operators, they're mixed:

```json
{"type":"COMPONENT_TYPE_CONNECTOR_AI","title":"Instill Model"}
{"type":"COMPONENT_TYPE_CONNECTOR_AI","title":"OpenAI"}
{"type":"COMPONENT_TYPE_CONNECTOR_APPLICATION","title":"Numbers Protocol"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"Pinecone"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"Redis"}
{"type":"COMPONENT_TYPE_OPERATOR","title":"JSON"}
{"type":"COMPONENT_TYPE_OPERATOR","title":"Image"}
{"type":"COMPONENT_TYPE_CONNECTOR_AI","title":"Archetype AI"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"BigQuery"}
{"type":"COMPONENT_TYPE_CONNECTOR_AI","title":"Stability AI"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"Website"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"REST API"}
{"type":"COMPONENT_TYPE_OPERATOR","title":"Text"}
{"type":"COMPONENT_TYPE_OPERATOR","title":"Base64"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"Google Search"}
{"type":"COMPONENT_TYPE_CONNECTOR_DATA","title":"Google Cloud Storage"}
{"type":"COMPONENT_TYPE_CONNECTOR_APPLICATION","title":"Slack"}
{"type":"COMPONENT_TYPE_CONNECTOR_AI","title":"Hugging Face"}
```

@donch1989